CONTRIBUTING.md

```diff
@@ -3,45 +3,55 @@
 
-If you'd like to contribute code, here are some things to keep in mind.
 
-* If your changes require any table modifications, don't forget to modify the
-  SQL/0000*.sql file(s) with your changes for new installs, and also
-  include a patch for existing projects to apply to get your changes of which
-  should be placed in the corresponding SQL/VERSIONNUMBER/ directory.
-* Include a test for any new module in the modules/MODULENAME/test/
+## Some Things To Keep In Mind
+
+* If your changes require any table modifications:
+    1. Modify the `SQL/0000*.sql` file(s) with your changes. These patches are applied during the LORIS install.
+    2. Include a patch to be used by existing projects. These should go in the `SQL/New_patches/` directory. 
+    3. For SQL patches that are _optional_ (e.g. those that perform some cleanup), place them in `SQL/Cleanup_patches/`.
+* Include a test for any new module in the `modules/MODULENAME/test/`
   directory. You can look at other modules for examples of how to write tests.
-  If you have questions, feel free to mail the mailing list.
-* Add your new tests to get auto-run by Travis in the .travis.yml to make sure that
-  other people don't accidentally break your module.
-* Check out our Coding Standards in the [docs/ directory](https://github.com/aces/Loris/tree/master/docs) and also our [Code Review Checklist](https://github.com/aces/Loris/wiki/Code-Review-Checklist) in the [GitHub Wiki](https://github.com/aces/Loris/wiki) and the Pull Request guidelines Readme in this directory. 
-* Try and make sure you run PHP codesniffer using the standards file in
-  docs/LorisCS.xml before sending any pull request, otherwise the Loris tests may
-  fail and we won't be able to merge your pull request.
-* Keep in mind that people are currently using Loris so try and make any changes
-  backwards-compatible with existing installations. If you must change something
-  in a non-backwards-compatible way, document it in your pull request description and
-  tag it with "Caveat For Existing Projects" so that we know that the change needs
-  to be mentioned in release notes. Non-backwards-compatible changes should be sent
-  to the next major release (e.g. 17.X -> 18.0) while backwards-compatible changes should
-  be sent to the next minor release (e.g. 17.1.0 -> 17.2.0) and backwards-compatible bug
-  fixes should be sent to the next minor release update (e.g. 17.1.0 -> 17.1.1)
-
-## Ways To Get Started
-
-If you're looking for ideas for a way to contribute but don't know where to get
-started, some ideas to get you started:
-
-* You can browse some of our public [Issues](https://github.com/aces/Loris/issues)
+* Add your new automated tests to TravisCI in the `.travis.yml`. 
+* Make sure you run PHP codesniffer using the standards file in
```

Comment:
should it be changed to incorporate the command to run phpcs? eg. 

> Make sure you run PHP codesniffer using the standards file in `docs/LorisCS.xml` by running `vendor/bin/phpcs --standard=docs/LorisCS.xml <path_to_changed_files>` before sending any pull request, otherwise the automated tests will fail.

Reply:
or atleast a reference to https://github.com/aces/Loris/blob/12135e9d606706c11ff88652e7c28d32af922b46/docs/CodingStandards#L21